content/en/docs/contributing/localization.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Site localization
+description:
+  Guidance on creating and maintaining site page in non-English localizations.
+linkTitle: Localization
+weight: 20
+---
+
+{{% pageinfo color="warning" %}}
+
+🚧 This DRAFT page is under active development. 🚧
+
+{{% /pageinfo %}}
+
+This website uses Hugo's [multilingual framework] to support page localizations.
+English is the default language, with US English as the default (implicit) localization.
+A growing number of other localizations are supported, as can be seen from the languages
+dropdown menu in the top nav.
+
+## Keeping track of localized page drift {#track-changes}
+
+One of the main challenges of maintaining localized pages is identifying when
+the corresponding English language pages have been updated. This section
+explains how we handle this.
+
+### The `default_lang_commit` front-matter field
+
+When a localized page is written, such as `content/zh/<some-path>/page.md`, this
+translation is based on a specific [`main` branch commit][main] of the
+corresponding English language version of the page at
+`content/en/<some-path>/page.md`. Every localized page identifies this commit in
+the page's front matter as follows:
+
+```markdown
+---
+title: Your localized page title
+...
+
+## default_lang_commit: <commit-hash-of-main-for-default-language-page>
```

Comment:
```suggestion
default_lang_commit: <commit-hash-of-main-for-default-language-page>
```

Without comment?

Reply:
Yes! Oops, I don't now how those comment characters got there.

Reply:
Thanks for catching that!